common4j/src/main/com/microsoft/identity/common/java/nativeauth/providers/NativeAuthResponseHandler.kt

```diff
@@ -237,7 +234,7 @@ class NativeAuthResponseHandler {
                 null,
                 EMPTY_RESPONSE_ERROR_ERROR_DESCRIPTION,
                 null,
-                null)
+                )
```

Comment:
can you fix the alignment of closing brace by making it consistent with the rest of the file?

Reply:
Fixed the alignment